metabrainz/templates/index/about.html

```diff
@@ -1,66 +1,66 @@
         regular updates of the data in order to help fund the project. We
         encourage all data users to contribute to the data gathering process so
-        that our data can be as comprehensive as possible.
+        that our data can be as comprehensive as possible.') }}
       </p>
       <p>
-        The MetaBrainz Foundation believes in <a href="{{ url_for('financial_reports.index') }}">
-        transparent finances</a> and has several <a href="{{ url_for('users.supporters_list') }}">
+        {{ _('The MetaBrainz Foundation believes in <a href="{{ url_for(\'financial_reports.index\') }}">
+        transparent finances</a> and has several <a href="{{ url_for(\'users.supporters_list\') }}">
         commercial supporters</a> who pay for timely and convenient access to
         data. The MetaBrainz Foundation is also supported by a number of
-        <a href="{{ url_for('index.sponsors') }}">sponsors</a> who provide
-        funds in order for the foundation to accomplish its goals.
+        <a href="{{ url_for(\'index.sponsors\') }}">sponsors</a> who provide
+        funds in order for the foundation to accomplish its goals.') }}
       </p>
     </div>
     <div class="col-sm-3">
       <img src="{{ url_for('static', filename='img/logos/metabrainz.svg') }}" class="hidden-xs" />
     </div>
   </div>
 
-  <h3>Board of Directors</h3>
-  Our board of directors consists of:
+  <h3>{{ _('Board of Directors') }}</h3>
+  {{ _('Our board of directors consists of:') }}
   <ul>
-    <li>Director Nick Ashton-Hart, Executive Director of the <a href="http://www.internet-ecosystem.org/people">Internet &amp; Digital Ecosystem Alliance</a></li>
-    <li>Director <a href="http://paulbennun.com/">Paul Bennun</a> of <a href="http://www.somethinelse.com/">Somethin' Else</a></li>
-    <li>Director <a href="http://craphound.com/">Cory Doctorow</a>, Sci-Fi author and editor of the <a href="https://boingboing.net/">Boing Boing</a> weblog</li>
-    <li>Director <a href="https://www.linkedin.com/in/sophie-goossens-9094759">Sophie Goossens</a> of <a href="http://www.august-debouzy.com/">August &amp; Debouzy</a></li>
-    <li>Director <a href="https://www.linkedin.com/pub/matthew-hawn/8/b74/8a">Matthew Hawn</a> of <a href="https://www.audionetwork.com/">Audio Network</a></li>
-    <li>Director &amp; Treasurer <a href="https://www.linkedin.com/in/rassamih">Rassami Hok Ljungberg</a> of the <a href="http://www.music4point5.com/">Music 4.5 seminar series</a> put on by <a href="http://2pears.com/">2 Pears</a></li>
-    <li>Executive Director <a href="http://mayhem-chaos.net/">Robert Kaye</a> of the MetaBrainz Foundation</li>
-    <li>Secretary <a href="https://freso.dk/">Frederik “Freso” S. Olesen</a>, Community Manager of the MetaBrainz Foundation</li>
+    <li>{{ _('Director Nick Ashton-Hart, Executive Director of the <a href="http://www.internet-ecosystem.org/people">Internet &amp; Digital Ecosystem Alliance</a>') }}</li>
```

Comment:
You shouldn't make assumptions about the sentence structure. It might be completely different in other languages. It also makes the work more difficult for translators.